tox.ini

```diff
@@ -99,6 +99,20 @@ extras =
 commands =
     pytest --basetemp={envtmpdir} --cov-report xml --cov-report html --cov=aicsimageio aicsimageio/tests/readers/extra_readers/test_ome_tiled_tiff_reader.py {posargs}
 
+[testenv:upstreams]
+passenv =
+    AWS_*
+    JAVA_HOME
+    CI
+setenv =
+    PYTHONPATH = {toxinidir}
+extras =
+    test
+deps =
+    distributed @ git+https://github.com/jakirkham/distributed@fix_6448
```

Comment:
I realize this is testing https://github.com/dask/distributed/pull/6449 at the moment, but just wanted to note that after we've converged on a fix over in `distributed`, we'll want to include both `dask` and `distributed` here 


```
dask @ git+https://github.com/dask/dask@main
distributed @ git+https://github.com/dask/distributed@main
```

Reply:
Yep, prior to testing the PR it was checking dask, distributed, and xarray upstreams.

Reply:
Ah, great 👍

Reply:
Going to wait to change this `aicsimageio` PR back until we are all done testing on that jakirkham / `distributed` branch